<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SiteSettingsBrowseDialog</class>
 <widget class="QDialog" name="SiteSettingsBrowseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Site Permissions</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Url:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="urlEdit">
       <property name="inputMethodHints">
        <set>Qt::InputMethodHint::ImhNoAutoUppercase|Qt::InputMethodHint::ImhUrlCharactersOnly</set>
       </property>
       <property name="placeholderText">
        <string>https://kde.org</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="urlComboBox"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="createButton">
     <property name="text">
      <string comment="Add or Modify">Set</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="indentation">
      <number>0</number>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Site</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Behaviour</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>50</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="askButton">
       <property name="text">
        <string>Ask</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="allowButton">
       <property name="text">
        <string>Allow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="denyButton">
       <property name="text">
        <string>Deny</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="defaultButton">
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SiteSettingsBrowseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SiteSettingsBrowseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
